openwisp_radius/apps.py

```diff
@@ -14,12 +16,27 @@
 from .utils import load_model, update_user_related_records
 
 
-class OpenwispRadiusConfig(AppConfig):
+class OpenwispRadiusConfig(ApiAppConfig):
     name = 'openwisp_radius'
     label = 'openwisp_radius'
     verbose_name = 'Freeradius'
 
+    API_ENABLED = True
+    REST_FRAMEWORK_SETTINGS = {
+        'DEFAULT_THROTTLE_RATES': {
+            # Strict throttling Policy
+            'throttle_obtain_auth': default_or_test('100/hour', None),
+            'throttle_validate_auth': default_or_test('100/hour', None),
```

Comment:
> @nemesisdesign, I have assumed strict throttling policy to be 100/hour requests, and A little bit less strict as 250/hour requests, the rest of them as 400/hour. Is that Okay?

I'll defer this to Fed only.

Reply:
Can this default to no throttling?
It may cause issues in real world systems so it's safer if it's opt-in.

Reply:
Yes, I have set it to 'None' as default, also updated the documentation for it.

Reply:
I still see `100/hour`